<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialTesterMainWindow</class>
 <widget class="QMainWindow" name="SerialTesterMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gamenastix XBox Receiver (C) DiscoWTMH </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="receivingTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendingTextEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetReceivingButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Receiving</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>701</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="sendRequestModelMessageButton">
       <property name="text">
        <string>Send RequestModel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="sendCalibrateNPoseMessageButton">
       <property name="text">
        <string>Send CalibrateNPose</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="sendCalibrateSPoseMessageButton">
       <property name="text">
        <string>Send CalibrateSPose</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>701</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLineEdit" name="responseModelTimestamp"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Last Timestamp</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="frameTimestamp"/>
     </item>
     <item row="2" column="2">
      <widget class="QTextEdit" name="frameMessage"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="responseModelMessage"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ResponseModel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Frame</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="compassLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="compassLabel">
       <property name="text">
        <string>compass</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="joystickPreviewLeftCanvas">
       <property name="text">
        <string>joystickLeftCanvas</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="joystickPreviewRightCanvas">
       <property name="text">
        <string>joystickRightCanvas</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>701</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>From Supervisor:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="refreshAvailablePortsButton_FromSupervisor">
       <property name="text">
        <string>Refresh Available Ports:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="availablePortsComboBox_FromSupervisor"/>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="baudRateComboBox_FromSupervisor">
       <property name="currentText">
        <string>115200</string>
       </property>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>To XBox Pad:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="connectButton_FromSupervisor">
       <property name="text">
        <string>Connect Supervisor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="refreshAvailablePortsButton_ToXBoxPad">
       <property name="text">
        <string>Refresh Available Ports:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="availablePortsComboBox_ToXBoxPad"/>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="baudRateComboBox_ToXBoxPad">
       <property name="currentText">
        <string>115200</string>
       </property>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="connectButton_ToXBoxPad">
       <property name="text">
        <string>Connect XBoxPad</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLineEdit" name="compassStatus"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Compass Status:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="joystickPreviewLeftTriggerCanvas">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>51</width>
      <height>109</height>
     </rect>
    </property>
    <property name="text">
     <string>leftTriggerCanvas</string>
    </property>
   </widget>
   <widget class="QLabel" name="joystickPreviewRightTriggerCanvas">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>51</width>
      <height>109</height>
     </rect>
    </property>
    <property name="text">
     <string>rightTriggerCanvas</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
